tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -235,4 +237,231 @@ describe("E2E Testing for taqueria flextesa plugin",  () => {
         }
     })
 
-});
\ No newline at end of file
+});
+
+describe("E2E Testing for taqueria typechecker and simulator tasks of the flextesa plugin", () => {
+
+    beforeAll(async () => {
+        dockerName = "local"
+        await generateTestProject(taqueriaProjectPath, ["flextesa"]);
+        await exec(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`})
+    })
+
+    test('Verify that taqueria typechecker task can typecheck one contract under contracts folder', async () => {
+        try {
+            // 1. Copy contract from data folder to taqueria project folder
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts`);
+
+            // 2. Run taq typecheck
+            const {stdout, stderr} = await exec(`taq typecheck ${dockerName}`, {cwd: `./${taqueriaProjectPath}`});
+
+            // 3. Verify that it's well-typed and contains no errors
+            expect(stdout).toContain("Valid");
```

Comment:
Your assertion here is checking that valid is in the output but you have made a file for the output contents. Can you make the assertion check that the `oneRowTable` is what is output?

Reply:
Will come in the next commit